packages/wonder-blocks-dropdown/components/check-icons.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+/**
```

Comment:
The comment should be placed above the `export default class ...` so that it gets picked up by Styleguidist.